h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -70,13 +71,14 @@ export default function AnnotationCard({
         await updateModerationStatus(status);
         onStatusChange(status);
         setModerationSaveState({ type: 'saved' });
-      } catch {
-        // TODO Catching all errors generally for now, but we need to explicitly
-        //      handle conflict errors
-        setModerationSaveState({
-          type: 'error',
-          error: 'An error occurred updating the moderation status',
-        });
+      } catch (e) {
+        const isConflictError =
+          e instanceof APIError && e.response?.status === 409;
+        const error = isConflictError
+          ? 'The annotation has been updated by its author'
```

Comment:
What response is intended to be used if someone else _moderated_ the annotation since it was loaded? We're using HTTP 409 Conflict here to represent the annotation itself being updated by its author, but another moderator moderating the same annotation is also a kind of conflict. Is a different status used for that?